3.6/data-modeling-databases-working-with.md

```diff
@@ -138,8 +138,13 @@ change into the current database to the new database. Changing the current
 database must explicitly be requested by using the
 *db._useDatabase* method.
 
-The *options* attribute currently has no meaning and is reserved for
-future use.
+The *options* attribute can be used to set defaults for collections that will
+be created in the new database:
+
+* *sharding*: The sharding method to use. Valid values are: "" or "single
+* *replicationFactor*: Default replication factor. Special values include "satellite",
+  which will replicate the collection to every DB-server, and 1, which disables replication.
```

Comment:
```suggestion
  which will replicate the collection to every DB-server, and "1", which disables replication.
```

Reply:
It has to be a number or a special string value. Strings will not be converted to numbers.

Reply:
I understand. "replicationFactor" can be a string "satellite" or a numeric.
But the value for "minReplicationFactor" should always be numeric.
The change I suggest above is not about turning the `1` into a string, but to make the value `1` stand a bit more out from the rest of the text formatting-wise. We could also put it inside asterisks or backticks if you prefer, but it should stand out a bit from the rest of the text.